src/lib/hash/keccak_fips/keccak_fips.h

```diff
@@ -0,0 +1,118 @@
+/*
+* Keccak-FIPS
+* (C) 2010,2016 Jack Lloyd
+* (C) 2023 Falko Strenzke
+*
+* Botan is released under the Simplified BSD License (see license.txt)
+*/
+
+#ifndef BOTAN_KECCAK_FIPS_H_
+#define BOTAN_KECCAK_FIPS_H_
+
+#include <botan/hash.h>
+#include <botan/secmem.h>
+#include <string>
+
+namespace Botan {
+
+/**
+* KECCAK FIPS
+*/
```

Comment:
It would be great the condense the discussion about the two variants of Keccak in #3279 into some clarifying Doxygen remarks here and in `keccak.h`. I'm sure future users and contributors would be grateful. 😄

Reply:
I added clarifying documentation in `keccak_fips.h` which comes in the next commit. The problem with `keccak.h` is that I am not really sure what it implements. I suspect it is what is implemented in "KECCAK - Final Algorithm Package" at https://csrc.nist.gov/projects/hash-functions/sha-3-project – but I didn't verify. It might be worthwhile to do so, but currently I don't have the time for it nor does this have priority in our current project.